code/modules/mob/pulling.dm

```diff
@@ -5,7 +5,7 @@
 		return
 	if(!AM || !isturf(AM.loc))	//if there's no object or the object being pulled is inside something: abort!
 		return
-	if(!(AM.anchored))
+	if(!(AM.anchored) && !usr.incapacitated())
```

Comment:
`usr` is the wrong thing to use for this, just check `incapacitated()`. Also, move it into it's own if statement above.
```
if(incapacitated())
    return
if(!AM.anchored)
    ...
```

Reply:
Didn't mean to click on that...